File: FIGURAS PYTHON/CUADRADO.py
class Cuadrado:
    def __init__(self, lado):
        print("creating the new square with side: {c}" .format(c=lado))
        self.lado = lado
    def perimetro(self):
        return 4*self.lado
    def Area(self):
        return (self.lado*self.lado)
    
    
cuadrado_pequeño = Cuadrado(15)
cuadrado_mediano= Cuadrado(24)
cuadrado_grande = Cuadrado(114)

print("Perimetro")
print(cuadrado_pequeño.perimetro())
print(cuadrado_mediano.perimetro())
print(cuadrado_grande.perimetro())

print("Area:")
print(cuadrado_pequeño.Area())
print(cuadrado_mediano.Area())
print(cuadrado_grande.Area())

File: FIGURAS JAVA/Cuadrado.java
public class Cuadrado {
    private double Area;
    private double Side;
    public Cuadrado(double Side) {
        System.out.println("creating the new square with side:"+ Side);
        this.Side=Side;
    }
    public double perimetro() {return 4*this.Side;}
    public double Area() {return this.Side*this.Side;}

    public static void main(String[] args) {
        Cuadrado cuadrado_pequeño = new Cuadrado(15);
        Cuadrado cuadrado_mediano = new Cuadrado(24);
        Cuadrado cuadrado_grande = new Cuadrado(114);

        System.out.println("Perimetro");
        System.out.println(cuadrado_pequeño.perimetro());
        System.out.println(cuadrado_mediano.perimetro());
        System.out.println(cuadrado_grande.perimetro());

        System.out.println("Area:");
        System.out.println(cuadrado_pequeño.Area());
        System.out.println(cuadrado_mediano.Area());
        System.out.println(cuadrado_grande.Area());
    }
}